src/app.js

```diff
@@ -31,7 +31,14 @@ class Messenger extends EventEmitter {
   /*:: greetings: RegExp */
   /*:: help: RegExp */
   /*:: options: Object */
-  constructor({hookPath = '/webhook', linkPath = '/link', emitGreetings = true, cache} = {}) {
+  /*:: pages: Object */
+  constructor({
+      hookPath = '/webhook',
+      linkPath = '/link',
+      emitGreetings = true,
+      cache,
+      pages = {}
```

Comment:
In the README I added:

> `pages`: A map of page ids to page access tokens `{1029384756: 'ThatsAReallyLongStringYouGotThere'}`. Currently optional but config will migrate to this in the future

but any suggestions to make things better are welcome.